experimental/examples/bubblebench/common/src/index.ts

```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT License.
  */
 
-export * from "./view";
```

Comment:
These are also done in #12398 - should they be excluded here, or maybe that one should be closed?